src/Uno.UI.Runtime.Skia.Gtk/GtkDisplayInformationExtension.cs

```diff
@@ -30,9 +30,6 @@ public GtkDisplayInformationExtension(object owner, Gtk.Window window)
 	public uint ScreenWidthInRawPixels => (uint)_window.Display.GetMonitorAtWindow(_window.Window).Workarea.Width;
 
 	public float LogicalDpi
-		// GetLogicalDpi cannot be used here to get the actual
```

Comment:
Why was the comment removed here?

Reply:
@MartinZikmund `GetLogicalDpi` is deleted now. I find it confusing to have a comment referring to something that doesn't exist.